JavaScript/Markdown_To_HTML/README.md

```diff
@@ -0,0 +1,22 @@
+# Markdown To HTML
+
+This script converts a markdown file to its corresponding html file
+
+## Setup instructions
+- Replace the contents of the file `markdown.md` with markdown content of your choice. This will be the input for this script
+- Enter the following commands in the terminal:
+  - `cd Rotten-Scripts\JavaScript\Markdown_To_HTML`
+  - `npm install` to install the dependencies required
+  - `node script.js` to run the script
+
+## Output
+A file `html.html` will be generated as output on running the script
+
+### Input File Example:
+![Input](https://i.imgur.com/QVr2D0C.png)
+
+### Corresponding Output File Example:
+![Output](https://i.imgur.com/AAvwxr6.png)
```

Comment:
You can also show the HTML web output for some good test cases including tables etc. in markdown. If you want, you can also add some more content in the markdown example. It will be great :)

Reply:
I added a table